cmd/common/frontends/frontends.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022-2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package frontends
+
+import (
+	"context"
+
+	"github.com/inspektor-gadget/inspektor-gadget/internal/logger"
+)
+
+type Frontend interface {
```

Comment:
The part of the Frontend in this PR is pretty much reduced to its bare bones - that's why it just outputs stuff. I had an alternative one, where it would take the full screen and just split top and bottom part - top for normal output/events and bottom for debug/info/errors and also handled scrolling. The GUI thingy from the presentation works similarly to route the output, so also implemented the Frontend interface.

What name would you prefer?